setup/instance/apply.go

```diff
@@ -0,0 +1,57 @@
+package instance
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"net/http"
+	"time"
+)
+
+func (l *LocalInstance) Apply(wait bool) error {
+	if err := l.assertExists(); err != nil {
```

Comment:
When creating a new instance, the directory wouldn't exist so this assert would always fail when creating a new instance? I found this to be the case when testing locally using this branch. 

```
➜  setup git:(setup-refactor) ✗ go run main.go apply atlantis
[WARNING] Version not set. Using default values for 'source' and 'version' inputs
Layer0 instance 'atlantis' does not exist locally.
Have you tried running `l0-setup pull atlantis`?
exit status 1
```

Even if I commented out the above assert code, apply still fails with the below.
```
➜  setup git:(setup-refactor) ✗ go run main.go apply atlantis
[WARNING] Version not set. Using default values for 'source' and 'version' inputs
No configuration files found!

Apply requires configuration to be present. Applying without a configuration
would mark everything for destruction, which is normally not what is desired.
If you would like to destroy everything, please run 'terraform destroy' instead
which does not require any configuration files.
exit status 1
exit status 1
```

cc @diemonster